pkg/pillar/devicenetwork/devicenetwork.go

```diff
@@ -49,6 +50,12 @@ func makeDevicePortConfig(ports []string, free []string) types.DevicePortConfig
 		config.Ports[ix].IsMgmt = true
 		config.Ports[ix].Name = config.Ports[ix].IfName
 		config.Ports[ix].Dhcp = types.DT_CLIENT
+		for _, port := range ctx.DevicePortConfig.Ports {
```

Comment:
Are you better of making this a method of ctx.DevicePortConfig ?
    port, err := ctx.DevicePortConfig.PortByIfName(u)
    if err == nil {
        config.Ports[ix].WirelessCfg = port.WirelessCfg
    }

Reply:
sure. let me look into this.